recipes/green/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.11.0" %}
+
+package:
+  name: greem
+  version: {{ version }}
+
+source:
+  fn: green-{{ version }}.tar.gz
+  url:   https://github.com/CleanCut/green/archive/{{version}}.tar.gz
```

Comment:
Is there a reason you're using github and not the pypi version? Looks like [the default source line](https://github.com/conda-forge/staged-recipes/blob/ff0002dc6f5ff8a580ca0f58faaa9bec7c44df93/recipes/example/meta.yaml#L18) should work, once you add `{% set name = "green" %}` at the top.

Reply:
I prefer to use the repos version based tag. The sha256sum between both pypi and github is different.

I find this on pypi
0a64e7372079f57643e26e220d1d8ccecb26ba6416963d61e2ec20d2990eda98

Reply:
Yeah, they're different because they're packaging different files: the pypi one doesn't include the images in the readme and all that cruft. When the pypi releases are functional, the conda-forge default is to use that. Though it also doesn't include the license file in this case, so whatever.